### PR TITLE
Onboard containerized docker apps

### PR DESCRIPTION
**Pull request by BillWagner**
Load the second eBook into Docs.

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and reopening to restart the build

**Comment by BillWagner** [comment]
closing and reopening again.

**Comment by BillWagner** [comment]
Removed the WIP label.  This is ready for review.  

cc @JRAlexander @mairaw

**Comment by JRAlexander** [comment]
I will review this afternoon

**Comment by BillWagner** [comment]
closing and reopening to start a new build.

File: docs/standard/architecture-containerized-liefecycle/docker-terminology.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+title: Docker terminology | Microsoft Docs 
+description: Containerized Docker Application Lifecycle with Microsoft Platform and Tools
+keywords: Docker, Microservices, ASP.NET, Container
+author: CESARDELATORRE
+ms.author: wiwagn
+ms.date: 09/21/2017
+---
+# Docker terminology
+
+This section lists terms and definitions with which you should become familiar with before delving deeper into Docker (for further definitions, see the extensive [glossary](https://docs.docker.com/v1.11/engine/reference/glossary/) provided by Docker at <https://docs.docker.com/v1.11/engine/reference/glossary/>):
```
Comment (JRAlexander) [inline docs/standard/architecture-containerized-liefecycle/docker-terminology.md]: This is an older url for the  glossary, the current one is here: https://docs.docker.com/glossary/, also it's listed twice with an extra parentheses at the end.

File: docs/standard/architecture-containerized-liefecycle/docker-terminology.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+-   **Docker Hub** A public registry to upload images and work with them. Docker Hub provides Docker image hosting, public or private registries, build triggers and web hooks, and integration with GitHub and Bitbucket.
+
+-   **Azure Container Registry** A public resource for working with Docker images and its components in Azure. This provides a registry that is close to your deployments in Azure and that gives you control over access, making it possible to use your Azure Active Directory groups and permissions.
+
+-   **Docker Trusted Registry (DTR)** A Docker registry service (from Docker) that you can install on-premises so that it resides within the organization's datacenter and network. It is convenient for private images that should be managed within the enterprise. Docker Trusted Registry is included as part of the Docker Datacenter product. For more information, go to [https://docs.docker.com/\
+    > docker-trusted-registry/overview/](https://docs.docker.com/docker-trusted-registry/overview/).
```
Comment (JRAlexander) [inline docs/standard/architecture-containerized-liefecycle/docker-terminology.md]: This URL is broken with a slash and linebreak.

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Overall, great job @BillWagner! Just a couple of very small things.

**Comment by BillWagner** [comment]
Thanks for the review @JRAlexander I've addressed all the feedback.

**Review by JRAlexander — APPROVED** [review]

**Pull request merged**